### PR TITLE
Feature/arc 253 ook interessant

### PR DESCRIPTION
**Pull request by BavoVdg**
- db integratie ook interessant
- updated back button to exclude detail pages

### CODE REVIEW

File: src/modules/media/hooks/get-media-similar.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export function useGetMediaSimilar(
+	id: string,
+	esIndex: string,
+	enabled = true
+): UseQueryResult<MediaSimilar> {
+	return useQuery(
+		[QUERY_KEYS.getMediaSimilar, { id }],
+		() => MediaService.getSimilar(id, esIndex),
+		{
+			keepPreviousData: true,
```
Comment (ian-emsens-shd) [inline src/modules/media/hooks/get-media-similar.ts:17]: Is dit geen default config?

File: src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx [hunk 2]
```diff
@@ -1,18 +1,19 @@
 import { Button, FlowPlayer, TabProps } from '@meemoo/react-components';
 import clsx from 'clsx';
+import { toLower } from 'lodash';
```
Comment (ian-emsens-shd) [inline src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx:3]: Vermijd `lodash`, gebruik `lodash-es` of `toLowerCase`

File: src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx [hunk 3]
```diff
@@ -132,6 +158,31 @@ const ObjectDetailPage: NextPage = () => {
...
+											asDate(hit._source.schema_date_published)
+									  )
+									: undefined
+							})`,
+							description: hit._source.schema_description || '',
+							// thumbnail: hit._source.schema_thumbnail_url,
+							id: hit._source.schema_identifier,
+						},
+					};
+				})
```
Comment (ian-emsens-shd) [inline src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx:182]: separate map function for legibility; `const mapSimilarData = () ...`

File: src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx [hunk 4]
```diff
@@ -387,32 +445,22 @@ const ObjectDetailPage: NextPage = () => {
...
+									{!!similar.length && (
+										<Metadata
+											className="u-px-32"
+											metadata={[
+												{
+													title: t('trefwoorden'),
+													data: mapKeywordsToTagList(mediaInfo.keywords),
+												},
+												{
+													title: 'Ook interessant',
```
Comment (ian-emsens-shd) [inline src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx:457]: i18n

File: src/styles/pages/_object-detail.scss [hunk 5]
```diff
@@ -161,6 +161,10 @@ $metadata-collapsed-width: 36vw;
 		}
 	}
 
+	&__similar-link {
+		text-decoration: none;
+	}
```
Comment (ian-emsens-shd) [inline src/styles/pages/_object-detail.scss:166]: Replace with https://github.com/viaacode/hetarchief-client/blob/release/v0.7.0/src/styles/utilities/_text.scss#L11-L13

**Review by ian-emsens-shd — CHANGES_REQUESTED** [review]

**Review by ian-emsens-shd — APPROVED** [review]

**Review by bertyhell — APPROVED** [review]

**Pull request merged**